data/card_catalogue.json

```diff
@@ -748,177 +790,97 @@
-            "flavour": "By Failing to prepare, you are preparing to fail.",
-            "image": "assets/core/card/illustration/Planned Strike.png"
-          }
-        },
-        {
-          "title": "Premeditated Strike",
-          "upgrade": "Cold Blooded Strike",
-          "tier": "green",
-          "details": {
-            "rules": "If you have 14 cards in hand, deal 20 damage to a character in melee range and stun them for 1 turn. Otherwise, deal 10",
-            "flavour": "I've been thinking about this one for a long time.",
-            "image": "assets/core/card/illustration/Premeditated Strike.png"
-          }
-        },
-        {
-          "title": "Cold Blooded Strike",
-          "upgrade": "",
-          "tier": "red",
-          "details": {
-            "rules": "If you have 14 cards in hand, deal 40 damage to a character in melee range and stun them for 1 turn. Otherwise, deal 20",
-            "flavour": "Man is the cruelest animal.",
-            "image": "assets/core/card/illustration/Cold Blooded Strike.png"
-          }
-        },
-        {
-          "title": "Desperate Strike",
-          "upgrade": "Anguished Strike",
-          "tier": "grey",
-          "details": {
-            "rules": "Deal 3 damage to a character in melee range. If you're at 10 or less life, deal 6 instead.",
-            "flavour": "An animal is most dangerous when cornered",
-            "image": "assets/core/card/illustration/Desperate Strike.png"
-          }
-        },
-        {
-          "title": "Anguished Strike",
-          "upgrade": "Tormented Strike",
-          "tier": "blue",
-          "details": {
-            "rules": "Deal 6 damage to a character in melee range. If you're at 10 or less life, deal 10 instead.",
-            "flavour": "We should forgive our enemies, but not before they are hanged",
-            "image": "assets/core/card/illustration/Anguished Strike.png"
-          }
-        },
-        {
-          "title": "Tormented Strike",
-          "upgrade": "Wretched Strike",
-          "tier": "green",
-          "details": {
-            "rules": "Deal 10 damage to a character in melee range. Knock them back X squares, where X is your missing health.",
-            "flavour": "Show them how it feels",
-            "image": "assets/core/card/illustration/Tormented Strike.png"
-          }
-        },
-        {
-          "title": "Wretched Strike",
-          "upgrade": "",
-          "tier": "red",
-          "details": {
-            "rules": "Deal 10 damage to a character in melee range and curse them until end of combat. Whenever a cursed character targets you with an attack, you may recast this spell.",
-            "flavour": "Every wrong shall be revenged, one hundred fold",
-            "image": "assets/core/card/illustration/Wretched Strike.png"
-          }
-        },
```

Comment:
These are just dull

Reply:
I think we should keep them, picking one up in a session isn't necessarily dull; either you're set on cards and you'll stash it for an upgrade or downgrade (which you're likely to do with most that you pick up) or you're low on cards and it stands to alter the course of the session (as you're probably in trouble). 

The real place they stand out is character creation though, they really invite a good backstory in.

Reply:
Moving this into a ticket